vip-workflow.php

```diff
@@ -62,8 +70,16 @@
 require_once VIP_WORKFLOW_ROOT . '/modules/shared/php/util.php';
 require_once VIP_WORKFLOW_ROOT . '/modules/shared/php/core-hacks.php';
 
-// Modules
+// Modules - Telemetry
+if ( class_exists( '\Automattic\VIP\Telemetry\Tracks' ) ) {
```

Comment:
We should do more than this imo. We incorporate the same `is_wpvip_site` check that we have on the block data api. That way, it would only activate this class, and send telemetry data if its an actual VIP site with a valid ID that we can use.